.travis.yml

```diff
@@ -1,10 +1,7 @@
 language: php
 
 php:
-  # 5.3.0 is not supported by Travis
-  - 5.3.3
-  - 5.3
-  - 5.4
+  - 5.5.9
```

Comment:
That's the version that comes with ubuntu 14.04. It's also kind of pointless to allow 5.5.0 since that's older than the lots of 5.4.x releases.